sdk/storage/azure-storage-datamovement/inc/azure/storage/datamovement/tasks/download_blob_to_file_task.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "azure/storage/datamovement/task.hpp"
+#include "azure/storage/datamovement/datamovement_options.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs { namespace _detail {
+
+  struct DownloadBlobToFileTask final : public Storage::_internal::TaskBase
+  {
+    DownloadBlobToFileTask(
+        Storage::_internal::TaskType type,
+        Storage::_internal::Scheduler* scheduler,
+        const Blobs::BlobClient& source,
+        const std::string& destination)
+        : TaskBase(type, scheduler), Context(std::make_shared<TaskContext>(source, destination))
+    {
+    }
+
+    struct TaskContext
+    {
+      explicit TaskContext(Blobs::BlobClient source, std::string destination)
+          : Source(std::move(source)), Destination(std::move(destination))
+      {
+      }
+      std::string Destination;
+      Blobs::BlobClient Source;
```

Comment:
Initialize members in the order they're defined

Reply:
Updated